### PR TITLE
1 a home page

### PR DESCRIPTION
**Pull request by elizlatkova**
home page is ready

### CODE REVIEW

File: src/components/AllBreweriesHome/AllBreweriesHome.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react';
+import BreweryInterface from "../BreweryCard/BreweryCard";
+
+export default function AllBrieweriesHome() {
+    const [breweries, setBreweries] = useState([]);
+    useEffect(() => {
+        fetch('https://api.openbrewerydb.org/v1/breweries?fbclid=IwZXh0bgNhZW0CMTAAAR1QKdPcqOwBI51QmYcnitDSFfKOnHlOLDSbxXadLJZgJJWfQ797QPTltHA_aem_ZmFrZWR1bW15MTZieXRlcw')
```
Comment (kalinadukova) [inline src/components/AllBreweriesHome/AllBreweriesHome.tsx]: You can export the request in a separate folder (api for example). It is good practice to keep the requests in a single folder location and not scatter them all around the app

File: src/components/AllBreweriesHome/AllBreweriesHome.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                return res.json();
+            })
+            .then((data) => {
+                console.log(data);
+                setBreweries(data);
+            });
+    }, []);
+
+    return (
+        <div className="allBrieweriesHome">
```
Comment (kalinadukova) [inline src/components/AllBreweriesHome/AllBreweriesHome.tsx]: css classes should follow the kebab case naming convension - https://smartwp.com/kebab-case/ or the BEM convension - https://css-tricks.com/bem-101/

File: src/components/BreweryCard/BreweryCard.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import "./BreweryCard.css"
+
+export default function BrieweryInterface({brewery}:{brewery:any}) {
```
Comment (kalinadukova) [inline src/components/BreweryCard/BreweryCard.tsx]: rename BrieweryInterface to BreweryCard - try to add the type of the brewery. Types should be exported in a separate folder and then imported wherever you want them

File: src/components/Navbar/Navbar.tsx [hunk 4]
```diff
@@ -4,12 +4,12 @@ import "./Navbar.css";
...
+        <nav className="navbarContainer">
             <img src={logo} alt="Logo" />
             <div className="navbarItemsRight">
-                <NavLink to="/"><span className="navbarItem">Home</span></NavLink>
-                <NavLink to="/random"><span className="navbarItem">Random brewery</span></NavLink>
-                <NavLink to="/favorites"><span className="navbarItem">Favorites</span></NavLink>
+                <NavLink to="/" className="navbarLink"><span className="navbarItem">Home</span></NavLink>
+                <NavLink to="/random" className="navbarLink"><span className="navbarItem">Random brewery</span></NavLink>
+                <NavLink to="/favorites" className="navbarLink"><span className="navbarItem">Favorites</span></NavLink>
             </div>
```
Comment (kalinadukova) [inline src/components/Navbar/Navbar.tsx:13]: A good practice is to use the Prettier extension to format the code. In every project that I have been on, so far it is used. You can check how to set it up in VSCode (there is an extension for it) and format every file of your code. If you don't manage to do that, I will help

**Review by kalinadukova — CHANGES_REQUESTED** [review]

File: src/components/AllBreweriesHome/AllBreweriesHome.tsx (hunk 2 above)
Comment (elizlatkova) [reply src/components/AllBreweriesHome/AllBreweriesHome.tsx]: i choose the kebab case and changed all class names

File: src/components/AllBreweriesHome/AllBreweriesHome.tsx (hunk 1 above)
Comment (elizlatkova) [reply src/components/AllBreweriesHome/AllBreweriesHome.tsx]: i think that i understood you and made a new folder named "api", then i moved the file and it's corresponding css file there in  a separate folder, from now on all files that include fetch will be found there

**Pull request merged**